sql/parse/parse.go

```diff
@@ -177,6 +177,12 @@ func convert(ctx *sql.Context, stmt sqlparser.Statement, query string) (sql.Node
 			return nil, err
 		}
 		return convertDDL(ctx, query, ddl.(*sqlparser.DDL))
+	case *sqlparser.MultiAlterDDL:
+		multiAlterDdl, err := sqlparser.ParseStrictDDL(query)
```

Comment:
You know, we should really get rid of this "strict" distinction some point, kinda dumb

Reply:
Yeah seems largely unnecessary